services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -125,9 +124,9 @@ export const FileUpload = (props: FileUploadProps) => {
     if (labelIdle) {
       return labelIdle;
     }
-    const fileTypeDisplayString = listedFileTypes
-      ? listedFileTypes.join(", ")
-      : Object.keys(acceptedFileTypesMap).join(", ");
+    const fileTypeList = listedFileTypes ?? Object.keys(acceptedFileTypesMap);
+    const fileTypeDisplayString =
+      fileTypeList.slice(0, -1).join(", ") + ", and " + fileTypeList.slice(-1);
```

Comment:
putting in the "and" to the filetypes list

Reply:
🎨